README.md

```diff
@@ -5,23 +5,21 @@
 
 ## What is CNI?
 
-CNI, the _Container Network Interface_, is a project to help configure network interfaces for Linux application containers.
-CNI concerns itself with setting up network connectivity of containers and removing allocated resources when the container is deleted.
+CNI, the _Container Network Interface_, specifies how to write plugins that configure network interfaces in Linux containers.
```

Comment:
I think we should try to give a summary in this first sentence instead of pointing out the spec. The other parts have become as important for users/developers as the spec has, right?